src/libcore/num/flt2dec/mod.rs

```diff
@@ -529,7 +629,12 @@ pub fn to_shortest_exp_str<'a, T, F>(mut format_shortest: F, v: T,
 /// 826 bytes of buffer should be sufficient for `f64`. Compare this with
 /// the actual number for the worst case: 770 bytes (when `exp = -1074`).
 fn estimate_max_buf_len(exp: i16) -> usize {
-    21 + ((if exp < 0 { -12 } else { 5 } * exp as i32) as usize >> 4)
+    21 +
+    ((if exp < 0 {
+        -12
+    } else {
+        5
+    } * exp as i32) as usize >> 4)
```

Comment:
The original expressions was overly complicated, for sure, but this reformatting is not helping.